canonical-web-and-design/maas-ui
# Set up preferences views

Done:
- Add preferences routes, nav etc. Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/1523

QA:
- Click your username in the top left (probably "admin").
- You should see the preferences nav.
